### DISCUSSION TITLE
General setup question - Which calendar address to use?

### DISCUSSION THREAD

**Discussion by tcdrake76** [derekantrican/GAS-ICS-Sync#438] [Q&A]
My apologies, I just can't seem to figure out what I am missing during the setup. I believe I have everything setup correctly but am not seeing the sync occur. 

I want to sync my sons baseball schedule from GameChanger. The URL from GC is webcal://api.team-manager.gc.com/ics-calendar-documents/user/814370e4-78ce-4569-9c2a-e3930279f8a6.ics?teamId=aa2c5421-f5aa-4ce5-bde6-1298eb2796e4&token=26e820870f47e36a81581fe80ed514873415a393bdaf9b871062a527c83261d3

I setup a calendar in Google calendar tied to the script with the name baseball. 

In the script I have the following...
  ["icsUrl1", "webcal://api.team-manager.gc.com/ics-calendar-documents/user/814370e4-78ce-4569-9c2a-e3930279f8a6.ics?teamId=aa2c5421-f5aa-4ce5-bde6-1298eb2796e4&token=26e820870f47e36a81581fe80ed514873415a393bdaf9b871062a527c83261d3" , "baseball"],

I also tried....
  ["webcal://api.team-manager.gc.com/ics-calendar-documents/user/814370e4-78ce-4569-9c2a-e3930279f8a6.ics?teamId=aa2c5421-f5aa-4ce5-bde6-1298eb2796e4&token=26e820870f47e36a81581fe80ed514873415a393bdaf9b871062a527c83261d3" , "baseball"],

Am I using the wrong URL?

**Comment by jonas0b1011001** [comment]
Did you resolve this and close the discussion on purpose?

**Comment by tcdrake76** [comment]
I did, as soon as I hit submit I noticed the calendar synced. I'm sorry for
the confusion.
